google-apis-generator/lib/google/apis/generator/patch.rb

```diff
@@ -0,0 +1,33 @@
+require 'google/apis/discovery_v1'
+
+# Extend the DiscoveryV1 API classes with additional fields for code generation,
+# supporting features which are not present in the schema itself.
+module Google
```

Comment:
One more thing I recommend doing is, wrap this in a check for if the `DiscoveryV1` code already includes this because it got updated. You could, for example, check to see whether RestMethod already defines the `api_version` method.

Reply:
I added logic for that, but I haven't been able to test that last part well.
I am unable to reload the generator module after creating a 'custom' version of `DiscoveryV1` within my test file. 

I tried splitting into separate files as well, but that didn't resolve it -- I assume it is persistent across the `toys ci` run. Please take a look at my previous commits for more info.

Reply:
Don't worry too much about it. This was going to be hard to test regardless, and it's expected to be a one-time event, so a permanent regression test on it won't be that important. From eyeballing it, it looks okay to me.